airbyte-integrations/connectors/source-mailjet-mail/source_mailjet_mail/spec.yaml

```
@@ -0,0 +1,21 @@
+documentationUrl: https://docsurl.com
+connectionSpecification:
+  $schema: http://json-schema.org/draft-07/schema#
+  title: Mailjet Mail Spec
+  type: object
+  required:
+    - username
+    - password
+  additionalProperties: true
+  properties:
+    username:
```

```suggestion
    api_key:
```
would be better call `api_key` and `api_key_secret`?